### PR TITLE
Preliminary Review

### LINKED ISSUE #2: Preliminary Review
- Review Chapters 1-5
- Fix any typos or language issues
- Small additions or suggestions

### PR DESCRIPTION
**Pull request by wendtke**
Address #2 for Chapters 1-5

### CODE REVIEW

**Review by SmogDr — APPROVED** [review]

File: index.Rmd [hunk 1]
```diff
@@ -114,5 +122,4 @@ online through the CSU library.
 
 ## Acknowledgements
 
-Most of the introductory material for this book were adapted from Professor [Brooke Anderson's R Programming Coursebook](https://geanders.github.io/RProgrammingForResearch/), to whom I owe thanks not only for the material but for the many helpful discussions.  I would also like to acknowledge [John Tukey](http://rsbm.royalsocietypublishing.org/content/49/537.full.pdf+html), one of the pioneers of exploratory data analysis, and the creators of the [NIST Engineering Statistics Handbook](https://doi.org/10.18434/M32189), from which I have drawn many techniques. 
-
+Most of the introductory material for this book were adapted from Professor [Brooke Anderson's R Programming Coursebook](https://geanders.github.io/RProgrammingForResearch/), to whom I owe thanks not only for the material but for the many helpful discussions. I would also like to acknowledge [John Tukey](http://rsbm.royalsocietypublishing.org/content/49/537.full.pdf+html), one of the pioneers of exploratory data analysis, and the creators of the [NIST Engineering Statistics Handbook](https://doi.org/10.18434/M32189), from which I have drawn many techniques.
```
Comment (SmogDr) [inline index.Rmd]: change 'material' to 'materials' for SV agreement.

**Review by SmogDr — APPROVED** [review]

**Review by SmogDr — APPROVED** [review]

**Comment by wendtke** [comment]
@SmogDr It looks like most of the merge conflicts are coming from automatic changes to binary files (e.g., re-render png). They would probably be manually resolved by cleaning up and/or deleting the automatically generated images. Instead, I am looking into [rebasing](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) as the safer merge option. I've never done it before, so I am reading up on it to make sure I do it right. Fingers crossed.

**Comment by SmogDr** [comment]
I don’t think you can do any serious damage at this point.  Go for it when you feel ready.

--
John Volckens
Professor, Mechanical Engineering
Colorado State University
he, him, his
jv.colostate.edu / @Smogdr

**Comment by wendtke** [comment]
I resolved the merge conflicts with the binary files manually from master to branch, rebased my branch with your master changes on chapter 8, and then merged successfully. But then I broke the master page build by accident. Fixing that now.
________________________________

**Pull request merged**